features/FEATURE_BOOTLOADER_SOURCES/Jenkinsfile

```diff
@@ -0,0 +1,283 @@
+//----------------------------------------------------------------------------
+//   The confidential and proprietary information contained in this file may
+//   only be used by a person authorised under and to the extent permitted
+//   by a subsisting licensing agreement from ARM Limited or its affiliates.
+//
+//          (C) COPYRIGHT 2016 ARM Limited or its affiliates.
+//              ALL RIGHTS RESERVED
+//
+//   This entire notice must be reproduced on all copies of this file
+//   and copies of this file may only be made by a person if such person is
+//   permitted to do so under the terms of a subsisting license agreement
+//   from ARM Limited or its affiliates.
+//----------------------------------------------------------------------------
+
+repoName = "mbed-bootloader-internal"
```

Comment:
Not a Jenkins expert.. does this +  "checkout scm" below, mean that Jenkins tests will still run on https://github.com/ARMmbed/mbed-bootloader-internal ?

Reply:
This file should be removed completely.